jujugui/static/gui/src/app/components/basic-table/basic-table.js

```diff
@@ -124,6 +128,8 @@ BasicTable.propTypes = {
       // The extra classes to apply to the column.
       classes: PropTypes.arrayOf(PropTypes.string)
     }).isRequired).isRequired,
+    // A value provided to the filter method.
+    filterValue: PropTypes.any,
```

Comment:
What is this used for? Leftover?

Reply:
This a value that can be use to supply values to the filter method.

Reply:
ISTM that the filter predicate takes the whole row, and could be in charge to decide what field to use to decide whether to include the row or not. It seems we are forcing to use one specific field while any could be used by the predicate.